`test_uv_install_strips_pythonpath` fails without `uv`

```
___________________ test_uv_install_strips_pythonpath[False] ___________________
[gw0] sunos5 -- Python 3.9.25 /data/builds/ul-mass-rebuild/components/python/build/build/amd64-3.9/.tox/py39/bin/python

mocker = <pytest_mock.plugin.MockerFixture object at 0x7fffacdf9970>
monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x7fffacbb1fd0>

    def test_uv_install_strips_pythonpath(
        mocker: pytest_mock.MockerFixture,
        monkeypatch: pytest.MonkeyPatch,
    ) -> None:
        monkeypatch.setenv('PYTHONPATH', '/some/leaky/path')
        run_subprocess = mocker.patch('build.env.run_subprocess')
>       with build.env.DefaultIsolatedEnv(installer='uv') as env:

tests/test_env.py:46: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
../prototype/i386/usr/lib/python3.9/vendor-packages/build/env.py:131: in __enter__
    self._env_backend.create(self._path)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <build.env._UvBackend object at 0x7fffac813e80>
path = '/tmp/build-env-w7lqzn42'

    def create(self, path: str) -> None:
        import venv
    
        self._env_path = path

        try:
            import uv

            self._uv_bin = uv.find_uv_bin()
        except (ModuleNotFoundError, FileNotFoundError):
            uv_bin = shutil.which(os.environ.get('UV') or 'uv')
            if uv_bin is None:
                msg = 'uv executable not found'
>               raise RuntimeError(msg) from None
E               RuntimeError: uv executable not found

../prototype/i386/usr/lib/python3.9/vendor-packages/build/env.py:379: RuntimeError
```

--- a/docs/changelog/1062.misc.rst
+++ b/docs/changelog/1062.misc.rst
@@ -1,0 +1,1 @@
+Skip ``test_uv_install_strips_pythonpath`` with missing ``uv`` - by :user:`mtelka`

--- a/tests/test_env.py
+++ b/tests/test_env.py
@@ -37,6 +37,7 @@ def test_make_extra_environ_overrides_pythonpath() -> None:
         assert env._env_backend.scripts_dir in extra['PATH']
 
 
+@pytest.mark.skipif(MISSING_UV, reason='uv executable not found')
 def test_uv_install_strips_pythonpath(
     mocker: pytest_mock.MockerFixture,
     monkeypatch: pytest.MonkeyPatch,

